project-addons/flask_middleware_connector/models/account_invoice.py

```diff
@@ -0,0 +1,19 @@
+from openerp import models, fields, api
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = "account.invoice"
+
+    orders = fields.Char('Orders', compute='get_orders', readonly=True, store=True)
+
+    @api.multi
+    @api.depends('invoice_line')
+    def get_orders(self):
+        orders = ''
+        for invoice in self:
+            for order in invoice.sale_order_ids:
+                if orders.find(order.name) == -1:
```

Comment:
Toda venta debería tener name es un campo obligatorio, por lo que te puedes ahorrar este find

Reply:
El find lo uso para ver si el pedido ya está en el string de orders y así no repetirlo. Tal vez podría hacerse de otras formas más eficientes, pero al final siempre tengo que comprobar si existe para saber si lo añado o no.

Reply:
El "orders = ''" debería estar dentro del for, ya que si entra con varioas ids al compute las últimas estarían sumando las ventas de todas las anteriores, respecto a lo que comentas, en una one2many no se pueden repetir los objetos, por lo que no debería hacer falta